lib/graphql/schema/warden.rb

```diff
@@ -170,7 +171,7 @@ def visible_abstract_type?(type_defn)
       end
 
       def visible_possible_types?(type_defn)
-        @schema.possible_types(type_defn).any? { |t| visible_type?(t) }
+        @schema.possible_types(type_defn, @context).any? { |t| visible_type?(t) }
```

Comment:
🤔 unrelated, but I wonder if this method should call through to `#possible_types`, eg 

```suggestion
        possible_types(type_defn).any?
```

Reply:
Yah it seems like possible types already does what this function does, but memoized? I think we could just about remove `visible_possible_types`. The only question I have is iof there is a case where the context can change but the results still be stored in memory, so we get stale data?

The other option is to just make this a bit cleaner by calling `visible_possible_types?` from inside of `possible_types` so we don't have the duplicate code.